unit1/02_diffusion_models_from_scratch.ipynb

```diff
@@ -1719,7 +1712,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.13"
+   "version": "3.10.8"
```

Comment:
maybe i'm too new to coding to ask this but why are you changing the python3 ver?